config/locales/en.yml

```diff
@@ -8,6 +8,14 @@ en:
   invalid_page_parameters: "The '#/page[page]' and '#/page[per_page]' parameter values must be a valid positive number"
   invalid_data_structure: correct json data structure required. See API docs for reference
   schedule_invalid_for_course: Selected schedule is not valid for the course
+  cannot_change_cohort: "You cannot change the '#/cohort' field"
```

Comment:
As above; can we stick these in `errors/models` and reuse existing translations where we already have them